src/Disqus.php

```diff
@@ -77,39 +46,19 @@ public function __construct(ContainerInterface $container, $apiKey, $secretKey =
         }
     }
 
-    /**
-     * @param string $shortname
-     * @param array  $options
-     * @param string $fetch
-     *
-     * @return string
-     */
-    public function fetch($shortname, array $options, $fetch = 'threads/listPosts')
+    public function fetch(string $shortname, array $options, string $fetch = 'threads/listPosts'): array
     {
         $this->shortname = $shortname;
 
         $options = $this->setOptions($options);
 
         $url = $this->buildUrl($options, $fetch);
-        if ($this->container->has('knp_zend_cache.manager')) {
-            $cache = $this->container->get('knp_zend_cache.manager');
-            $cache = $cache->getCache($this->container->getParameter('knp_disqus.cache.'.$shortname));
-            $key   = sha1($url);
-            if (false === ($content = $cache->load($key))) {
-                $content = json_decode($this->httpRequest($url), true);
-
-                // we cache, even if we have a bad response
-                // sometimes disqus goes down (request times out), and we don't want to keep making
-                // this hanging request each time - just don't server-side load the comments for now
-                $cache->save($content, $key);
-            }
-        } else {
-            $content = json_decode($this->httpRequest($url), true);
-        }
+
+        $content = $this->httpRequest($url);
```

Comment:
You removed caching and I think this was a correct thing to do.

To make it possible/easy for users to add cache, we should:

A) Create a `DisqusInterface` and make this class implement it. That will allow for decoration
B) Document how you could decorate this service in your app and add caching.